packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -187,14 +187,20 @@ test('close vault', async t => {
       message,
     },
   );
-  t.like(wd.getLatestUpdateRecord(), {
+  const updateRecord = wd.getLatestUpdateRecord();
+  t.like(updateRecord, {
     updated: 'offerStatus',
     status: {
       id: 'close-insufficient',
       numWantsSatisfied: 1, // trivially true because proposal `want` was empty.
-      error: `Error: ${message}`,
     },
   });
+  if (updateRecord.updated === 'offerStatus') {
```

Comment:
This condition is for type narrowing? Worth a comment. 

But with a quoted value could the simpler like remain?

Reply:
Yes, the quoted value would make this unnecessary.